lib/activerecord-mysql-structure/active_record/structure_sql_sanitizer.rb

```diff
@@ -17,20 +20,34 @@ def self.sorted_columns?
       @sorted_columns
     end
 
+    # When set to true, the sanitizer will sort the index definitions of each
+    # table. This will help keep a consistent index order in the outputted
+    # structure file. The indices will be sorted alphabetically, and redundant
+    # index_type clauses will be stripped
+    def self.sorted_indices=(val)
```

Comment:
What do you think about adding YARD docs to these methods? I think it'd be nice if we did more of that in v3.